image_create.sh

```diff
@@ -211,29 +211,29 @@ END_SFDISK
 			cat << END_SFDISK | "$sfdisk_tool" --no-tell-kernel "$output"
 label: gpt
 start=- size=16MiB type=21686148-6449-6E6F-744E-656564454649
-start=- size=256MiB type=uefi
+start=- size=256MiB type=uefi uuid=CB7F0CDA-721E-41A6-B8A9-06F21910AA07
```

Comment:
Perhaps the UUID should be configurable via an env var (like the `gpt_type` for the root partition) with a default value if not set? Also might be nice to also have an option to set the partition UUID for the root partition.